.azure-devops-android-tests.yml

```diff
@@ -25,7 +25,10 @@ jobs:
       BUILD_SOURCESDIRECTORY: "$(build.sourcesdirectory)"
       BUILD_ARTIFACTSTAGINGDIRECTORY: "$(build.artifactstagingdirectory)"
 
-
+  - bash: cp $(build.sourcesdirectory)/src/SamplesApp/SamplesApp.Droid/obj/Release/g/XamlCodeGenerator/* $(build.artifactstagingdirectory)/XamlCodeGenerator/
```

Comment:
You may not want to keep that, but you can add it to the troubleshooting steps in the documentation

Reply:
I discussed with Benji, we'd like to have it optional and off by default, right now it will only copy when it fails. But maybe having it commented out and part of docs would be good.

Reply:
It's more about the fact that it's only doing it for one target. What was the issue that you faced that needed this ?

Reply:
Build fails on a source generation issue only on Android....dumping the generated code into the artifacts I'll be able to investigate that specific code. I see your point, so you simply want to document that script for troubleshooting purposes. Agreed. I'll make sure to remove that step once I can resolve the issue.

Reply:
It has been removed and the troubleshooting docs have been updated